app/src/androidTest/java/com/github/polypoly/app/map/MapViewModelTest.kt

```diff
@@ -12,7 +12,7 @@ import org.junit.Test
 
 @OptIn(ExperimentalCoroutinesApi::class)
 @InternalCoroutinesApi
-class GameViewModelTest {
+class MapViewModelTest {
```

Comment:
Not completely sure to understand why reverting to the old name, I'm assuming because it is now not the case that the ViewModel is dedicated to the GameActivity class ?

Reply:
I changed the name because I use it in a part which is not the game and that the interior of this class concerns only the map in general, there is nothing specific to the game